src/frontend/src/pages/GanttPage/GanttPageWrapper.tsx

```diff
@@ -75,6 +87,18 @@ const GanttPageWrapper: FC = () => {
 
   useEffect(() => {
     const transformProjectToGanttObject = (project: Project): Task => {
+      project.workPackages.sort((a, b) => {
```

Comment:
Looks great - only thing is that there's a bit of code duplication here. Can we abstract this to one function that we can use in both cases?